RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.11.2
+
+- fixed: `git machete` now correctly infers remote for fetching of branch when the branch is associated with more than one remote
```

Comment:
```suggestion
- fixed: `git machete` now correctly infers remote for fetching of branch when the branch is found in more than one remote
```

As _associated_ would suggest `branch.<branch>.remote` config being in place... while here it's just about a branch with the given name being present in multiple remotes (which can theoretically be a coincidence!)